<h5>What is BadMut?</h5>
<p class="text-justify">BadMut (Bad Mutation; we like how it resembles
    "bad mood") is a meta-estimator designed to distinguish between neutral and
    disease-causing amino acid substitutions (i.e. nonsynonimous single
    nucleotide variations – nsSNVs). It is based on a machine-learning
    technique commonly known as deep learning, that is it uses deep (multilayer)
    artificial neural networks to make predictions.</p>

<h5>What is a meta-estimator?</h5>
<p class="text-justify">Untangling the connections between variation
    and phenotypic traits remains the greatest challenge of functional genomics,
    thus a lot of effort has been put into developing the means to infer
    possible damage of novel (phenotypically uncategorised) nsSNVs by employing
    machine-learning. As a result, over the past decade many algorithms have
    been developed for predicting deleteriousness. In order to make predictions
    these tools encode variants using multiple quantitative and qualitative
    features of proteins and their sequences. An ensemble of such tools can be
    further used to get an integrated prediction - this is exactly what
    meta-estimators (aka ensemble scores and meta-scores) do.</p>

<h5>What kind of information BadMut uses to make predictions?</h5>
<p class="text-justify">Being a meta-estimator, BadMut uses the output
    of other scores as its main source of information. In particular, BadMut
    integrates SIFT, Polyphen-2, LRT, MutationTaster, MutationAssessor, FATHMM,
    PROVEAN, MetaSVM, MetaLR, GERP++, phyloP, phastCons, SiPhy. It also uses
    population statistics (allele frequency) generated by the 1000 Genomes
    project. BadMut takes all the information from
    <a href="https://sites.google.com/site/jpopgen/dbNSFP">dbNSFP v3.1</a>.
</p>

<h5>What can one use BadMut for?</h5>
<p class="text-justify">BadMut can be used to prioritise
    likely-damaging novel amino acid substitutions in silico prior to expensive
    laboratory testing. Although it doesn't analyse noncoding regions of the
    genome, unlike most other tools it provides near-complete coverage of the
    human exome.</p>

<h5>How to interpret the output?</h5>
<p class="text-justify">Unlike many other tools BadMut doesn't assign
    the substitutions it processes to different classes (e.g.
    pathogenic/neutral). Instead, it returns the probability of clinical
    significance. One thus can apply a binary cutoff threshold to separate
    neutral and pathogenic substitutions. Selecting a value for the threshold is
    a question of balancing sensitivity (the ability to find damaging mutations)
    and specificity – lower thresholds result in greater sensitivity and lower
    specificity and vice versa.</p>
