<h5>Exp-miBR annotator</h5>
<p class="text-justify"> is a web-tool that allows to annotate a VCF-file or single
    position with experimentally confirmed microRNA binding regions (Exp-miBR)
    and their features.
</p>
<br>

<h5>Motivation</h5>
<p class="text-justify">microRNAs are small noncoding RNAs, that accomplish an essential post-transcriptional
    regulation of gene expression. Previously, we showed that commonly used microRNA-mRNA interactions
    predicting software differ from each other substantially, while had a great discrepancy with microRNAs’
    binding regions identified in experiments [1]. Experimentally identified microRNA binding regions
    could be promising as a basis for further queries into not only the fundamental aspects of gene
    regulation but may uncover some new mechanisms caused diseases.
</p>
<br>

<h5>What is exp-miBRs?</h5>
<p class="text-justify"> We have aggregated various experimental data on human microRNA-mRNA
    interactions to identify reliable microRNA binding regions:
</p>
<ul class="list-unstyled">
    <li>Two existing experiments that had developed and optimized technique protocol to obtain
        full human microRNA-mRNA interactome: CLASH [2] and CLEAR-CLIP [3]. </li>
    <li>79 AGO2-CLIP-seq data from 9 different cell lines.</li>
</ul>
<p>We have extracted from these microRNA binding regions only highly reliable regions (Exp-miBRs) that
had a subsequence of length L=10, whereas each nucleotide (position) in this subsequence had been supported
by at least n=2 different datasets or chimeras. <br>
In total, we revealed 46,8 thousand of experimentally confirmed microRNA binding regions,
that were located in all parts of mRNAs, including noncoding regions.
</p>
<br>

<h5>Exp-miBRs features:</h5>
<p class="text-justify"> We characterized each Exp-miBR by the following parameters:
</p>
<ul class="list-unstyled">
    <li><b>Region information:</b> Genome strand +/- (MIRNA_STRAND), gene name (MIRNA_GENES),
             type of mRNA part (MIRNA_REGION).</li>
    <li><b>Supported dataset characteristics:</b> amount (MIRNA_NDATASET) and list (MIRNA_DATASET)
        of supported experiments (IDs from www.ncbi.nlm.nih.gov/geo) and their corresponding cell
        lines (MIRNA_LINES, MIRNA_NLINES).</li>
    <li><b>Interacted miRNAs</b> (that were detected in CLASH or CLEAR-CLIP experiment as a part of chimeras):
        list of interacted microRNAs (if accessible) (MIRNA).</li>
</ul>
<br>
<h5>How it works?</h5>
<h3>Starter on the service:</h3>
<p class="text-justify">There are a few different modes to analyze nucleotide variants location in exp-miBR:
</p>
<ul class="list-unstyled">
    <li><b>Process a single point:</b> allows you to submit a single point at a time.
        While you can make as many of these submissions as you want. If you have more than a dozen of
        points to process, you should consider the VCF submission form instead.</li>
    <li><b>Process a VCF query:</b> Allow you to upload and process any VCF file (v4.0-4.2),
        though your upload can be no more than 20MB. The file can be compressed by any gzip-compatible
        method (for instance, gzip itself or bgzip). To limit workload, our system only processes the
        first 500k lines from a submitted VCF file (excluding the header lines).
        The output is a bgzipped VCF v4 file. Please, check that your vcf file has a correct
        heading form (you can copy it from example vcf file below).</li>
    <li><b>Download track:</b> Allow you to download all exp-miBRs with their characteristics as a
        separate file. There are separate files for both genome versions (hg19 and hg38).
        Each line in file is corresponded to one exp-miBR.</li>
</ul>
<br>
<h3>Example of processing</h3>
<p class="text-justify"><b>Input vcf file contains the following variants:</b></p>
<p class="text-justify">SOME HEADINGS</p>
<p class="text-justify">#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO </p>
<p class="text-justify">1	40073961	.	.	.	.	.	.</p>
<p class="text-justify">3	68194389	.	G	A	.	.	.</p>
<p class="text-justify">7	43876072	.	.	.	.	.	.</p>

<p class="text-justify"><b>The output vcf file will be extended by new fields in INFO (to all variants
    that are located in exp-miBRS) and their descriptions in heading section.
</b></p>
<p class="text-justify">SOME HEADINGS</p>
<p class="text-justify">##INFO=ID=MIRNA_STRAND,Number=.,Type=String,Description="A comma-separated list of miRNAs"</p>
<p class="text-justify">##INFO=ID=MIRNA_NDATASET,Number=.,Type=Integer,Description="Number of datasets"</p>
<p class="text-justify">##INFO=ID=MIRNA_DATASET,Number=.,Type=String,Description="A comma-separated list of datasets"</p>
<p class="text-justify">##INFO=ID=MIRNA_NLINES,Number=.,Type=Integer,Description="Number of cell lines"</p>
<p class="text-justify">##INFO=ID=MIRNA_LINES,Number=.,Type=String,Description="A comma-separated list of cell lines"</p>
<p class="text-justify">##INFO=ID=MIRNA,Number=.,Type=String,Description="A comma-separated list of miRNAs"</p>
<p class="text-justify">##INFO=ID=MIRNA_REGION,Number=.,Type=String,Description="A comma-separated list of region descriptors"</p>
<p class="text-justify">##INFO=ID=MIRNA_GENES,Number=.,Type=String,Description="A comma-separated list of genes"</p>
<p class="text-justify">#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO </p>
<p class="text-justify">1	40073961	.	.	.	.	.
    MIRNA_STRAND=-;MIRNA_NDATASET=3;MIRNA_DATASET=GSE32113%GSM796037,GSE32113%GSM796038,GSE50452%CLASH;MIRNA_NLINES=2;
    MIRNA_LINES=BC-1,HEK293;MIRNA=hsa-miR-320a;MIRNA_REGION=UTR3;MIRNA_GENES=PPT1</p>
<p class="text-justify">3	68194389	.	G	A	.	.	.</p>
<p class="text-justify">7	43876072	.	.	.	.	.	MIRNA_STRAND=-;MIRNA_NDATASET=2;
    MIRNA_DATASET=GSE28865%GSM714644,GSE28865%GSM714646;MIRNA_NLINES=1;MIRNA_LINES=HEK293;MIRNA_REGION=UTR3;
    MIRNA_GENES=URGCP</p>
<br>
<h3>Interpretation:</h3>
<ol class="list-unstyled">
    <li><b>Variant chr1 40073961:</b> is located in exp-miBR that was supported
        in 3 (MIRNA_NDATASET=3) experiments (MIRNA_DATASET=GSE32113%GSM796037,GSE32113%GSM796038,GSE50452%CLASH).
        These three datasets are corresponded to two different cell lines (MIRNA_NLINES=2; MIRNA_LINES=BC-1,HEK293).
        There is one known interacted microRNA (MIRNA=hsa-miR-320a). This exp-miBR is located in 3 UTR region
        (MIRNA_REGION=UTR3) of PPT1 gene (MIRNA_GENES=PPT1).</li>
    <li><b>Variant chr3 68194389:</b> is not located in any exp-miBRs.</li>
    <li><b>Variant chr7 43876072:</b> is located in exp-miBR that was supported in 2
        (MIRNA_NDATASET=2) experiments (MIRNA_DATASET= GSE28865%GSM714644,GSE28865%GSM714646).
        These two datasets are corresponded to only one cell line (MIRNA_NLINES=1;MIRNA_LINES=HEK293).
        Due to the fact that all supported datasets are AGO-CLIP there is no information about interacted microRNA.
        This exp-miBR is located in 3 UTR region (MIRNA_REGION=UTR3) of PPT1 gene (MIRNA_GENES=URGCP).</li>
</ol>
<p class="text-justify">Full information about supported experiments could be find at
    <a class="text-info" href="http://www.ncbi.nlm.nih.gov/geo">NCBI GEO</a> by the provided GSE and GSM ids.
</p>
<br>

<h3>Download:</h3>
<p class="text-justify">Example of input vcf file with correct headings:
    <a class="text-info" href="https://drive.google.com/open?id=1QYTmKeVi_P7dCCCriFe_7VAPQFK3k8AA">
        Example_input.vcf (634B)</a></p>
<p class="text-justify">Example of output vcf file:
    <a class="text-info" href="https://drive.google.com/open?id=1pQaWJezZKKU8mWTL8uNv-hvofTg95Q0s">
        Example_output.vcf (2KB)</a></p>
<p class="text-justify">Download all high confidence microRNA binding regions along with the characteristics of each of
    these regions as tsv file (reference of human genome: <b>hg19</b>)
    <a class="text-info" href="https://drive.google.com/open?id=130ER2qFoFWZIq67TwNSIlFrmcJJLMmB8">
        Exp-miBRS_track_information_hg19.tsv (6,5MB)</a></p>
<p class="text-justify">Download all high confidence microRNA binding regions along with the characteristics of each of
    these regions as tsv file (reference of human genome: <b>hg38</b>)
    <a class="text-info" href="https://drive.google.com/open?id=13FU7nSHZWPEMuj8bKONDTLlj9-Oame6G">
        Exp-miBRS_track_information_hg38.tsv (6,6MB)</a></p>
<br>

<h5>How to cite:</h5>
<p class="text-justify">While the paper is under review, please cite
    <a class="text-info" href="https://www.biorxiv.org/content/10.1101/675694v1">pre-print version</a></p>
<p class="text-justify">Plotnikova, Olga, Ancha Baranova, and Mikhail Skoblov. "Comprehensive analysis of
    human microRNA-mRNA interactome." bioRxiv (2019): 675694. doi: https://doi.org/10.1101/675694</p>
<br>

<h5>Reference:</h5>
<ol class="list-unstyled">
    <li>O.M.Plotnikova, M. Y. Skoblov (2018) Efficiency of the miRNA–mRNA interaction prediction
        programs, Molecular Biology, 52.3:467–477.</li>
    <li>Helwak A. et al. (2013) Mapping the human miRNA interactome by CLASH reveals
        frequent noncanonical binding, Cell, 153.3:654–665.</li>
    <li>Moore M. J. et al. (2015) miRNA–target chimeras reveal miRNA 3′-end pairing as a major
        determinant of Argonaute target specificity, Nature communications, 6:8864.</li>
</ol>